### ISSUE TITLE
fields that are changed by RENDERING_MAPPINGS are not synchronised

### ISSUE THREAD

**Issue by hhaensel** [GenieFramework/Stipple.jl#275]
Currently, names of fields are replaced by Stipple.RENDERING_MAPPINGS when a model is rendered. Also when a field is newly pushed from the backend, the name is correctly translated.
However, for the the syncing to the backend there is no inverse translation. Particularly after loading StippleUI, a lot of fieldnames are translated.

There are two possible ways out:
- include a reverse translation
- don't translate at all

I, personally, was very surprised to see translation happening at all. I think it is can be quite confusing, particularly if you write templates.

In some cases it even breaks functionality, e.g. when "selected" is translated to "v-model:selected". (I knew it was not functioning for more than a year, but only found out the reason today...)

@essenciary How do we proceed?

**Comment by essenciary** [comment]
I don't fully understand - can you provide a MWE with more details on what to look for?

**Comment by hhaensel** [comment]
Take the follwoing MWE:

```julia
@app HH begin                                                                                          
    @in keepcolor = true                                                                             
end

ui() = toggle("test", :keepcolor)

route("/") do                                                                                          
    global model                                                                                       
    model = @init HH                                                                                   
    page(model, ui) |> html                                                                            
end

model.keepcolor[]
# keeps being true, independent of the toggle position
```
The reason is that "keepcolor" is translated to "keepColor"
```julia-repl
julia> render(HH())[:data]
Dict{String, Any} with 4 entries:
  "keepColor"    => true
  "isready"      => false
  "isprocessing" => false
  "fileuploads"  => Dict{AbstractString, AbstractString}()
```
During field updating we don't find the field "keepColor" and reject the incoming update request.

**Comment by hhaensel** [comment]
In case of "selected", the translation to "v-model:selected" doesn't make sense at all and the field is not exposed.

**Comment by essenciary** [comment]
Won't removing them be breaking? People might be using them in their apps.

**Comment by hhaensel** [comment]
It's breaking, but only for potential js expressions where people must have been aware of the julia_to_view translation. I doubt that people are using it, because the synching to the backend is not working. Someone would have complained.

**Comment by hhaensel** [comment]
I think it is sufficient to remove the addition of render mappings in StippleUI.

**Comment by hhaensel** [comment]
In this context I looked where we are using `julia_to_vue`.
I found a stange occurrence in https://github.com/GenieFramework/Stipple.jl/blob/master/src/Stipple.jl#L1097-L1098
```julia
attr_val = isa(v, Symbol) && ! startswith(string(k), ":") ? Stipple.julia_to_vue(v) : v
```
as this changes the value of an attribute pair. All other occurences address the key. Maybe this is just wrong?
```julia
julia> Stipple.attributes([:mykey => :menuoffset])
(var":mykey" = "menuOffset",)
```
Most probably we can just remove `julia_to_vue` completely, as the only thing it does is converting a Symbol to a string, which will happen anyhow if we json the final dict.
